src/main/java/com/rocketden/main/model/problem/Problem.java

```diff
@@ -36,7 +37,7 @@
      * is deleted or test cases are removed from this list, the test cases will also be deleted.
      * Setter is set to private to ensure proper use of addTestCase and removeTestCase methods.
      */
-    @OneToMany(mappedBy = "problem", cascade = CascadeType.ALL, orphanRemoval = true)
+    @OneToMany(mappedBy = "problem", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
Just out of curiosity how come this requires the eager fetch?

Reply:
To be honest, I did not research it past finding [this Stack Overflow post](https://stackoverflow.com/questions/22821695/how-to-fix-hibernate-lazyinitializationexception-failed-to-lazily-initialize-a); I was getting errors about LazyInitializationException as it related to the Problem class and its ProblemTestCase field, so this seemed to fix it.